emacs-live-py-mode/live-py-mode.el

```diff
@@ -146,22 +146,17 @@ both are relative to (point-min). Numbering starts at 1 for all
 When the source buffer is narrowed the trace buffer remains
 aligned but will not hide the part after the narrowing."
   (let* ((output-window (get-buffer-window live-py-trace-name))
-         (point-min-pos (point-min))
-         (scroll-margin 0)
-         (point-min-line-nr (if (= 1 point-min-pos)
-                                1
-                              ;; Compensate for narrowing.
-                              (save-restriction
-                                (widen)
-                                (line-number-at-pos point-min-pos)))))
+         (point-min-line-nr (count-lines 1 (point-min)))
+         (window-start-line-nr (+ point-min-line-nr window-start-line-nr -1))
+         (point-line-nr (+ point-min-line-nr point-line-nr -1)))
     (unless output-window
-      (live-py-create-output-window))
+      (live-py-create-output-window)
+      (set-window-buffer output-window live-py-trace-name))
     (with-selected-window output-window
       (goto-char 1)
-      (forward-line (+ point-min-line-nr window-start-line-nr -2))
+      (forward-line window-start-line-nr)
       (recenter-top-bottom 0)
-      (forward-line (- point-line-nr window-start-line-nr)))
-    (set-window-buffer output-window live-py-trace-name)))
```

Comment:
This move breaks

live-py-test/3-other-window/4-abandon-source-and-swap/edit
live-py-test/3-other-window/4-abandon-source-and-swap/move

What issue is the move expected to fix?